### DISCUSSION TITLE
umbraco 9: update ImageSharp dependency to version 2.x

### DISCUSSION THREAD

**Discussion by shearer3000** [umbraco/Umbraco-CMS#12147] [Features and ideas]
ImageSharp version 2 is RTM (currently v2.1 at time of writing). This version has support for WebP format which we would like to utilize in our umbraco website frontends once more (like we could in Umbraco 8 / Imageprocessor).

**Comment by nul800sebastiaan** [comment]
Please add to the existing discussion instead :) https://github.com/umbraco/Umbraco-CMS/discussions/11604

**Reply by nul800sebastiaan** [reply]
FYI: Additionally, ImageSharp.Web 2.0, with WebP support has not yet been released https://www.nuget.org/packages/SixLabors.ImageSharp.Web/
